flink-streaming-scala/src/main/scala/org/apache/flink/streaming/api/scala/DataStream.scala

```diff
@@ -441,6 +463,26 @@ class DataStream[T](stream: JavaStream[T]) {
    */
   def broadcast: DataStream[T] = asScalaStream(stream.broadcast())
 
+  /**
+    * Sets the partitioning of the [[DataStream]] so that the output elements
+    * are broadcasted to every parallel instance of the next operation. In addition,
+    * it implicitly creates a
```

Comment:
I think the "creates a" is outdated here because you can specify multiple descriptors. This should also be fixed in the Java API.